Fix rust compiler meeting info (#1087)
rust-lang/rustc-dev-guide: src/compiler-team.md

## Rust compiler meeting

The compiler team has a weekly meeting where we do triage and try to
generally stay on top of new bugs, regressions, and discuss important
things in general.
They are held on [Zulip][zulip-meetings]. It works roughly as follows:

- **Announcements, MCPs/FCPs, and WG-check-ins:** We share some
  announcements with the rest of the team about important things we want

>   everyone to be aware. We also share the status of MCPs and FCPs and we

Reviewer: ```suggestion
  everyone to be aware of. We also share the status of MCPs and FCPs and we
```

  use the opportunity to have a couple of WGs giving us an update about
  their work.
- **Check for beta and stable nominations:** These are nominations of things to
  backport to beta and stable respectively.
  We then look for new cases where the compiler broke previously working
  code in the wild. Regressions are important issues to fix, so it's
  likely that they are tagged as P-critical or P-high; the major
  exception would be bug fixes (though even there we often [aim to give
  warnings first][procedure]).
- **Review P-critical and P-high bugs:** P-critical and P-high bugs are
  those that are sufficiently important for us to actively track
  progress. P-critical and P-high bugs should ideally always have an
  assignee.
- **Check S-waiting-on-team and I-nominated issues:** These are issues where feedback from
  the team is desired.
- **Look over the performance triage report:** We check for PRs that made the

>     performance worse and try to decide if it's worth reverting the performance regression or if 

Reviewer: ```suggestion
    performance worse and try to decide if it's worth reverting the performance regression or if
```

>     the regression can be addressed in a future PR. 

Reviewer: ```suggestion
    the regression can be addressed in a future PR.
```

The meeting currently takes place on Thursdays at 10am Boston time
(UTC-4 typically, but daylight savings time sometimes makes things
complicated).

> The meeting is held over a "chat medium", currently on [zulip].

Reviewer: ```suggestion

```

We mentioned it on L35, this could be removed (and the link is broken anyway).

## Rust compiler meeting

The compiler team has a weekly meeting where we do triage and try to
generally stay on top of new bugs, regressions, and discuss important
things in general.
They are held on [Zulip][zulip-meetings]. It works roughly as follows:

> - **Announcements, MCPs/FCPs and WG-check-ins:** we share some

Reviewer 2: ```suggestion
- **Announcements, MCPs/FCPs and WG-check-ins:** We share some
```

> - **Announcements, MCPs/FCPs and WG-check-ins:** We share some

Reviewer 2: Totally optional, but we tend to like our Oxford comma ;)

```suggestion
- **Announcements, MCPs/FCPs, and WG-check-ins:** We share some
```

  announcements with the rest of the team about important things we want
  everyone to be aware. We also share the status of MCPs and FCPs and we
  use the opportunity to have a couple of WGs giving us an update about
  their work.
- **Check for beta and stable nominations:** These are nominations of things to
  backport to beta and stable respectively.
  We then look for new cases where the compiler broke previously working
  code in the wild. Regressions are important issues to fix, so it's
  likely that they are tagged as P-critical or P-high; the major
  exception would be bug fixes (though even there we often [aim to give
  warnings first][procedure]).

> - **Review P-critical/P-high bugs:** P-critical and P-high bugs are

Reviewer 2: ```suggestion
- **Review P-critical and P-high bugs:** P-critical and P-high bugs are
```

  those that are sufficiently important for us to actively track
  progress. P-critical and P-high bugs should ideally always have an
  assignee.
- **Check S-waiting-on-team and I-nominated issues:** These are issues where feedback from
  the team is desired.

> - **Look over the performance logs:** we check for PRs that made the

Reviewer 2: ```suggestion
- **Look over the performance logs:** We check for PRs that made the
```

> - **Look over the performance logs:** We check for PRs that made the
>     performance better in some sense and the ones that made it worser
>     and try to decide if it's worth reverting some regression or if it's
>     more work to be done to make things better.

Reviewer 3: ```suggestion
- **Look over the performance triage report:** We check for PRs that made the
    performance worse and try to decide if it's worth reverting the performance regression or if 
    the regression can be addressed in a future PR. 
```
